BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -691,9 +703,7 @@ namespace big
 							selected_color = color;
 							wheel_color = color;
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_EXTRA_COLOURS(player_vehicle, pearlescent, wheel_color);
-							});
+							VEHICLE::SET_VEHICLE_EXTRA_COLOURS(player_vehicle, pearlescent, wheel_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.